safe/gis/vector/multi_buffering.py

```diff
@@ -122,8 +123,9 @@ def multi_buffering(layer, radii, callback=None):
             callback(current=i, maximum=feature_count, step=processing_step)
 
     # We transfer keywords to the output.
-    buffered.keywords = layer.keywords
+    buffered.keywords = KeywordIO().read_keywords(layer)
```

Comment:
Hum no, we don't use KeywordIO inside the gis package as the layer should have monkey patching. We assume we have monkey patching everytime. Please keep the old line.

But I'm wondering why you care about keywords from the source layer. The layer shouldn't have keywords as the purpose of the multi-buffer tool is to create the hazard layer, so AFAIK the layer will never have keywords before this step.